toolbox/fdc3-for-web/fdc3-web-impl/src/handlers/HeartbeatHandler.ts

```diff
@@ -36,6 +40,8 @@ export class HeartbeatHandler implements MessageHandler {
 
   constructor(pingInterval: number = 1000, disconnectedAfter: number = 5000, deadAfter: number = 20000) {
     this.timerFunction = setInterval(() => {
+      //console.log(`Contexts: ${this.contexts.length} Last Heartbeats: `, this.heartbeatTimes())
```

Comment:
Should this line stay in?

Reply:
removed  - will of course generate endless ticking messages